features/step_definitions/common.rb

```diff
@@ -115,6 +115,33 @@
     table([[pattern]])
 end
 
+When /^the output of the :([a-z_]*?) client command in loop `(.+?)` should( not)? (contain|match) "(.+)"(?: (\d+) times)? with:$/ do |yaml_key, what, negative, match_type, pattern, times, table|
```

Comment:
This is really hard to read.

Reply:
@lihongyan1 not sure if this make it easier to use w/o documentation.  FYI, we already have a step to loop.  You can use this as a reference: https://github.com/openshift/verification-tests/blob/master/features/test/meta.feature